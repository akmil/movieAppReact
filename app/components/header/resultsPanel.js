import React from 'react';

export const ResultsCount = ({count}) => (
    <div className='results-count'>{count} items found</div>
);

export const ResultsSort = ({activeItem}) => (
    <div className={'results-sort-by'}>
        Sort by
        <ul className={'results-sort-by-list'}>
            <li>{`release date`}</li>
            <li className={`active`}>{`raiting`}</li> {/*TODO get from activeItem*/}
        </ul>
    </div>
);