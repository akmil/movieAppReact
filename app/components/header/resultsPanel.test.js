import React from 'react';
// import renderer from 'react-test-renderer';
import { ResultsCount, ResultsSort } from './resultsPanel';
import { shallow } from 'enzyme';

describe('resultsPanel', () => {
  it('should render without errors ResultsCount', () => {
    // const wrapper = renderer.create(<MainLayout/>)
    const wrapper = shallow(<ResultsCount/>);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render without errors ResultsSort', () => {
    // const wrapper = renderer.create(<MainLayout/>)
    const wrapper = shallow(<ResultsSort/>);
    expect(wrapper).toMatchSnapshot();
  });
});
