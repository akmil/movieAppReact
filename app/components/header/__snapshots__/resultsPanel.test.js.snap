// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resultsPanel should render without errors ResultsCount 1`] = `
<div
  className="results-count"
>
   items found
</div>
`;

exports[`resultsPanel should render without errors ResultsSort 1`] = `
<div
  className="results-sort-by"
>
  Sort by
  <ul
    className="results-sort-by-list"
  >
    <li>
      release date
    </li>
    <li
      className="active"
    >
      raiting
    </li>
     
  </ul>
</div>
`;
